lib/ansible/galaxy/role.py

```diff
@@ -30,6 +41,14 @@
 from distutils.version import LooseVersion
 from shutil import rmtree
 
+try:
+    import boto3
+except ImportError:
+    try:
```

Comment:
The [module checklist](http://docs.ansible.com/ansible/developing_modules.html#module-checklist) (and I know this isn't a module) suggest using 

```
try:
    import boto
    HAS_BOTO = True
except ImportError:
    HAS_BOTO = False
```
Then you can just use HAS_BOTO/HAS_BOTO3 rather than `try/except NameError`. 

But in this case it's a stylistic thing, and using common Ansible practices, not really a blocker.